pkg/controller/nodelifecycle/config/types.go

```diff
@@ -46,5 +46,5 @@ type NodeLifecycleControllerConfiguration struct {
 	// unhealthyZoneThreshold (no less than 3) of Nodes in the zone are NotReady
 	UnhealthyZoneThreshold float32
 	// tenant api-server URLs
-	TenantServers []string
+	TenantPartitionKubeConfigs []string
```

Comment:
Perfer single string as the code won't need change when we work on TP/RP auto discovery

Reply:
wont fix. single string is to the commandline  agrs, which is already A comma separated string. this array holds the kubeconfigs after the value from cmd line is parsed.